core/server/models/base/plugins/bulk-operations.js

```diff
@@ -52,9 +52,9 @@ async function editMultiple(knex, table, chunk, options) {
     await knex(table).whereIn('id', chunk).update(options.data);
 }
 
-async function delSingle(knex, table, id) {
+async function delSingle(knex, table, id, options) {
```

Comment:
should this be `options = {}`?

Reply:
I didn't add this because editSingle, editMultiple also don't have a default value. And since this is an internal method, we know that there will always be an options object passed to it because `bulkDestroy` already has the default.